### DISCUSSION TITLE
v1.5.0

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#4239] [Announcements]
## Windows App SDK 1.5.0 (1.5.240227000)

WinAppSDK 1.5.0 is the latest stable version of WinAppSDK providing new features and improvements to the platform.

**New features include:**
- XAML Islands runtime and shutdown updates
- The new WinUI Maps control and new SelectorBar control
- Labels in the CommandBarFlyout primary commands

**Additional updates include:**
- Support for .NET 8
- Improved debugging tools and source availability

**To see everything that's new and changed, see the full [Windows App SDK 1.5 release notes](https://aka.ms/windowsappsdk/1.5/1.5.0/release-notes).**  

**See the [WinAppSDK 1.5 milestone](https://github.com/microsoft/WindowsAppSDK/milestone/16?closed=1) and [WinUI 3 in WinAppSDK 1.5 milestone](https://github.com/microsoft/microsoft-ui-xaml/milestone/20?closed=1) for a list of issues addressed in this release.**

### Try it out
- Download the [1.5.240227000 NuGet](https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.5.240227000) package to use WinAppSDK 1.5 in your app.
- Download and update the [WinUI Gallery](https://www.microsoft.com/store/productId/9P3JFPWWDZRC) to see the WinUI 3 updates firsthand.

### Getting Started 

To get started using Windows App SDK to develop Windows apps, check out the following documentation:    
- [Install developer tools ](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment).     
- [Create a WinUI 3 app](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app).   
- [Continue your development journey](https://docs.microsoft.com/windows/apps/develop).   

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.5.0'>v1.5.0</a>.</em>

**Comment by cmmeyer358** [comment]
How can I install this in a yml file? Using this task:  
 - task: UseDotNet@2
    displayName: 'Install .NET SDK 1.5'
    inputs:
      packageType: 'sdk'
      version: '1.5.0'
      includePreviewVersions: true 
I have tried every combination of version reference--1.5, 1.5.0, 1.5.2, 1.5.24--but they all return an error saying version cannot be found or, in the case of 1.5, that the format is incorrect

**Reply by duncanmacmichael** [reply]
Hi, I'm sorry but it looks like you're trying to install the .NET SDK with this yml and not the Windows App SDK. To install the Windows App SDK, I'd recommend checking out the [Downloads page](https://learn.microsoft.com/windows/apps/windows-app-sdk/downloads) for the installer or to download the NuGet directly using the link in the announcement above.
